src/serac/physics/solid_mechanics.hpp

```diff
@@ -349,6 +349,16 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
    * @tparam MaterialType The solid material type
    * @tparam StateType the type that contains the internal variables for MaterialType
    * @param material A material that provides a function to evaluate stress
+   * @pre body_force must be a object that can be called with the following arguments:
+   *    1. `tensor<T,dim> x` the spatial coordinates for the quadrature point
```

Comment:
I believe this should be state, displacement gradient, parameters....

Reply:
Whoops, good point! This is the main source of the confusion: although `Functional` has a clear set of rules for arguments are handled, our physics modules have no consistency on what they expect from the q-functions.